server/controllers/stock/inventory/receipts/list.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const path = require('path');
+const BadRequest = require('../../../../lib/errors/BadRequest');
+
+let supportedRender = {
+  json : require('../../../../lib/renderers/json'),
+  html : require('../../../../lib/renderers/html'),
+  pdf  : require('../../../../lib/renderers/pdf')
+};
+
+// export the receipt object
+exports.build = build;
+
+/**
+ * @function build
+ * @desc build a report for inventory list of metadata
+ * @param {array} data inventory list of metadata
+ * @return {object} promise
+ */
+function build(data, request) {
+  'use strict';
+
+  let queryString   = request.query;
+  let defaultRender = 'pdf';
+  let template = path.normalize('./server/controllers/stock/inventory/receipts/list.handlebars');
```

Comment:
Sine the `defaultrender`, `template`, and `receiptOptions` never change, it is much better to define them outside of the function as constants.  Just like you did with `supportedRender`.

```js
const template = path.normalize('./server/controllers/stock/inventory/receipts/list.handlebars');
// other constants ...

function build(data, request) {
  // main function body ..
}
```
The advantages of this are:
 1. Less garbage collection, since NodeJS doesn't create a lot of variables and free them immediately after the function is called.
 2. A bit cleaner code - the variables that are constant and do not depend on the function parameters are not mixed in the function body.
 3. No chance of accidentally overwriting these variables at runtime.